src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdRepresentative.java

```diff
@@ -65,48 +76,4 @@ private static void updateContact(String caseRef, SscsCaseData existingCcdCaseDa
         existingCcdCaseData.getAppeal().getRep().setContact(contact);
     }
 
-    private static void updateRepresentativeSubscription(SscsCaseData gapsCaseData, SscsCaseData existingCcdCaseData) {
```

Comment:
Why do we remove this Rep subscription code?

Reply:
I've copied the code to another file and made it take functions so that it can update appellant, appointee and rep subscriptions.